cmd/kube-proxy/app/server_test.go

```diff
@@ -549,3 +554,9 @@ func (s *fakeProxyServerError) Run() error {
 		return fmt.Errorf("mocking error from ProxyServer.Run()")
 	}
 }
+
+// CleanUpAndExit runs in the specified ProxyServer.
+func (s *fakeProxyServerError) CleanUpAndExit() error {
+	time.Sleep(2 * time.Second)
```

Comment:
I don't think the sleep is necessary, as far as I can see we're not really testing async behavior.

Reply:
Updated.
Thanks